extraction/text_extractor.py

```diff
@@ -0,0 +1,41 @@
+"""This script checks whether DOM has text tag or not and
+creates and returns the Text object"""
+
+import bs4
+from extraction.data_models.text import Text
+
+TEXT_TAGS = {'p', 'span', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'code'}
+HEADING_TAGS = {'h1', 'h2', 'h3', 'h4', 'h5', 'h6'}
+
+
+def empty_text(_str):
+    """Returns true is null string"""
+    return not _str and not _str.strip()
+
+
+def not_important_text(node):
+    """Returns true if the tag has no important text"""
+
+    return 0 < len(node.get_text()) < 50 \
+        and node.name not in HEADING_TAGS
+
+
+def is_text(node):
+    """Validates if a tag is text tag and
+    returns the extracted data from the text tag in Text object"""
+
+    if isinstance(node, bs4.element.Tag):
+        text_data = node.get_text().strip()
+        if (node.name in TEXT_TAGS) \
+                and not empty_text(text_data) \
+                and not not_important_text(node):
```

Comment:
Is not_important_text() used anywhere else? If not, changing the method to is_important_text() would probably make this more readable.

Reply:
Makes sense! I will make the changes.